src/app/pie-chart/pie-chart.component.ts

```diff
@@ -11,7 +11,7 @@ export class PieChartComponent {
   }
 
   public chartType = 'pie';
-  url = 'http://10.3.4.114:4200/Gradle___Alor_Backend_war/rest/mdm/dist';
+  urlMoodDistGet = 'http://10.3.4.114:4200/Gradle___Alor_Backend_war/rest/mdm/dist';
```

Comment:
urlMoodDistGet is still a meaningless name. If I don't have any knowledge about that code/project I can't make up anything out of that name. What about urlMoodDistribution?